mrsbaylock/tests/test_eval_mgmt.py

```diff
@@ -191,42 +186,78 @@ def test_add_invalid_supp_section(self):
-        self.dept_details_admin_page.ignore(e)
+        self.dept_details_admin_page.mark_as_confirmed([e])
         self.dept_details_admin_page.wait_for_eval_rows()
-        self.dept_details_admin_page.click_ignored_filter()
-        assert e.status['ui'] in self.dept_details_admin_page.eval_status(e)
+        assert e.status.value['ui'] in self.dept_details_admin_page.eval_status(e)
 
-    def test_set_unmarked_status(self):
-        e = next(filter(lambda row: (row.status == EvaluationStatus.CONFIRMED), self.dept_1.evaluations))
-        self.dept_details_admin_page.unmark(e)
+    def test_set_confirmed_status_single(self):
+        e = next(filter(lambda row: (row.status is None), self.dept_1.evaluations))
+        self.dept_details_admin_page.click_edit_evaluation(e)
+        self.dept_details_admin_page.select_eval_status(e, EvaluationStatus.CONFIRMED)
+        self.dept_details_admin_page.click_save_eval_changes(e)
+        self.dept_details_admin_page.wait_for_eval_rows()
+        e.status = EvaluationStatus.CONFIRMED
+        assert e.status.value['ui'] in self.dept_details_admin_page.eval_status(e)
+
+    def test_set_ignored_status_bulk(self):
+        e = next(filter(lambda row: (row.status is None), self.dept_1.evaluations))
+        self.dept_details_admin_page.bulk_ignore([e])
+        self.dept_details_admin_page.wait_for_eval_rows()
+        self.dept_details_admin_page.select_ignored_filter()
+        assert e.status.value['ui'] in self.dept_details_admin_page.eval_status(e)
+
+    def test_set_ignored_status_single(self):
+        e = next(filter(lambda row: (row.status is None), self.dept_1.evaluations))
+        self.dept_details_admin_page.click_edit_evaluation(e)
+        self.dept_details_admin_page.select_eval_status(e, EvaluationStatus.IGNORED)
+        self.dept_details_admin_page.click_save_eval_changes(e)
+        self.dept_details_admin_page.wait_for_eval_rows()
+        e.status = EvaluationStatus.IGNORED
+        assert e.status.value['ui'] in self.dept_details_admin_page.eval_status(e)
+
+    def test_set_unmarked_status_bulk(self):
+        e = next(filter(lambda row: (row.status == EvaluationStatus.FOR_REVIEW), self.dept_1.evaluations))
+        self.dept_details_admin_page.bulk_unmark([e])
         self.dept_details_admin_page.wait_for_eval_rows()
-        self.dept_details_admin_page.click_ignored_filter()
         assert not self.dept_details_admin_page.eval_status(e)
 
+    def test_set_unmarked_status_single(self):
+        e = next(filter(lambda row: (row.status == EvaluationStatus.FOR_REVIEW), self.dept_1.evaluations))
+        self.dept_details_admin_page.click_edit_evaluation(e)
+        self.dept_details_admin_page.select_eval_status(e, EvaluationStatus.FOR_REVIEW)
+        self.dept_details_admin_page.click_save_eval_changes(e)
+        self.dept_details_admin_page.wait_for_eval_rows()
+        e.status = EvaluationStatus.FOR_REVIEW
```

Comment:
@pfarestveit Should the status here be FOR_REVIEW or UNMARKED?

Reply:
It should be "my brain is fried" status.  Thanks for catching @pauline2k .